src/main/java/frc/robot/Robot.java

```diff
@@ -29,6 +29,33 @@ public void robotInit() {
 
         coralArmCommand = new CoralArmCommand(coralArm);
         coralArm.setDefaultCommand(coralArmCommand);
+
+        algaeArm.setDefaultCommand(new RetractAlgaeArm(algaeArm));
+
+        Command checkIfLow = Commands.defer(()-> {
+            if(coralElevator.isRaised()){
+                return new LowerCoralElevator(coralElevator);
+            }
+            return Commands.none();
```

Comment:
actually, using `none` isn't a good option, because this command will end immediately and then be restarted every 20 ms.

Use `Commands.idle(coralElevator)`. This command won't end unless overridden.